tasks/0-setup.yaml

```diff
@@ -62,10 +62,11 @@ spec:
       default: quay.io/ibmgaragecloud/ibmcloud-dev:v2.0.4
     - name: js-image
       default: quay.io/ibmgaragecloud/node:lts-stretch
+    - name: ace-project
```

Comment:
Why do we have ace-project as input?
We should use this input if the user want to be explicit to set the value, they can specify it here, if not defined then fall back to on using `ls workspace` if workspace exists

Reply:
Complete